app/coffeescript/components/mixins/map_utils.coffee

```diff
@@ -3,12 +3,13 @@ define [ 'jquery' ], ( $ ) ->
   mapUtils = ->
     @defaultAttrs
       assetUrl: undefined
-      assetHostSelector:    'meta[name="asset_host"]'      
-      spinnerSelector:      '.spinner'
-      refinementsSelector:  '.pageInfo[name="refinements"]'
-      propertyNameParam:    'propertyname'
-      mgtcoidParam:         'mgtcoid'
-      propertyManagementRE: 'property-management'
+      assetHostSelector:      'meta[name="asset_host"]'
+      spinnerSelector:        '.spinner'
+      refinementsSelector:    '.pageInfo[name="refinements"]'
+      pinRefinementsSelector: '.pageInfo[name="pin_search_refinements"]'
```

Comment:
My preference would be to reuse the existing refinements meta tag if possible (I can't think of any reason it wouldn't be).